pkg/api/accesscontrol.go

```diff
@@ -276,15 +276,15 @@ func (hs *HTTPServer) declareFixedRoles() error {
 		Grants: []string{string(models.ROLE_VIEWER)},
 	}
 
-	localAnnotationsWriterRole := ac.RoleRegistration{
+	dashboardAnnotationsWriterRole := ac.RoleRegistration{
 		Role: ac.RoleDTO{
-			Name:        "fixed:annotations.local:writer",
-			DisplayName: "Local annotation writer",
+			Name:        "fixed:annotations.dashboard:writer",
+			DisplayName: "Dashboard annotation writer",
 			Description: "Update annotations associated with dashboards.",
 			Group:       "Annotations",
 			Version:     1,
```

Comment:
Commenting so that we don't forget: need to increase the version here :)

Reply:
Done thanks!